applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -58,4 +59,35 @@ struct le_audio_encoded_audio {
 	uint8_t num_ch;
 };
 
+int le_audio_tx_stream_stopped(uint8_t idx);
+
+int le_audio_tx_stream_started(uint8_t idx);
+
+int le_audio_tx_stream_sent(uint8_t idx);
+/**
+ * @brief	Allocates buffers and sends data to the controller
```

Comment:
```suggestion
 * @brief	Allocates buffers and sends data to the controller.
```

Reply:
Thanks @greg-fer, but no need to review draft PRs :)

Reply:
Now it's not draft anymore :)